resources/js/pages/public/index.tsx

```
@@ -4,7 +4,7 @@ import { Card, CardContent, CardHeader } from '@/components/ui/card';
 import AppLogoIcon from '@/components/app-logo-icon';
 import MarketingLayout from '@/layouts/marketing-layout';
 import { login } from '@/wayfinder/routes/index';
-import { create } from '@/wayfinder/routes/public/unit';
+import { create } from '@/wayfinder/routes/home/unit';
```

ESLint

[error] `@/wayfinder/routes/home/unit` import should occur before import of `@/wayfinder/routes/index`

(import/order)